esp/scm/ws_dfu.ecm

```diff
@@ -437,9 +437,10 @@ DFUSpaceResponse
     ESParray<ESPstruct DFUSpaceItem> DFUSpaceItems;
 };
 
-ESPrequest DFUFileViewRequest
+ESPrequest [nil_remove] DFUFileViewRequest
```

Comment:
When ESP receives a DFUFileViewRequest from HTPP call or SOAP call, ESP will parse the information from the request into a data object and forward data object to WsDFU.DFUFileView. The nil_remove means: if a member variable of DFUFileViewRequest is not in the request, ESP will not add it into the data object. For example, if the IncludeSuperOwner is not in an HTTP DFUFileViewRequest, the IncludeSuperOwner (<IncludeSuperOwner/>) will not be in the data object.